draft-ietf-quic-qpack.md

```diff
@@ -662,32 +663,32 @@ entries which are frequently referenced, both to avoid the need to resend the
 header and to avoid the entry in the table blocking the ability to insert new
 headers.
 
-### Dynamic Table Size Update {#size-update}
+### Dynamic Table Capacity Update {#capacity-update}
 
-An encoder informs the decoder of a change to the size of the dynamic table
-using an instruction which begins with the '001' three-bit pattern.  The new
-maximum table size is represented as an integer with a 5-bit prefix (see Section
-5.1 of [RFC7541]).
+An encoder informs the decoder of a change to the dynamic table capacity using
+an instruction which begins with the '001' three-bit pattern.  The new dynamic
+table capacity is represented as an integer with a 5-bit prefix (see Section 5.1
+of [RFC7541]).
 
 ~~~~~~~~~~ drawing
   0   1   2   3   4   5   6   7
 +---+---+---+---+---+---+---+---+
-| 0 | 0 | 1 |   Max size (5+)   |
+| 0 | 0 | 1 |   Capacity (5+)   |
 +---+---+---+-------------------+
 ~~~~~~~~~~
-{:#fig-size-change title="Maximum Dynamic Table Size Change"}
+{:#fig-capacity-update title="Dynamic Table Capacity Update"}
 
-The new maximum size MUST be lower than or equal to the limit determined by the
-protocol using QPACK.  A value that exceeds this limit MUST be treated as a
-connection error of type `HTTP_QPACK_ENCODER_STREAM_ERROR`.  In HTTP/3, this
-limit is the value of the SETTINGS_HEADER_TABLE_SIZE parameter (see
-{{configuration}}) received from the decoder.
+The new capacity MUST be lower than or equal to the limit described in
+{{maximum-dynamic-table-capacity}}.  On 1-RTT HTTP/3 connections, this limit is
```

Comment:
True for all connections -- the setting has an initial value of (0 , previous) and then gets updated when the SETTINGS frame is received.